### ISSUE TITLE
R_DD is not applied

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo-phi-normalizer#549]
Here is what I'm doing:

```
$ cat bar.phi
{⟦ m ↦ ⟦ x ↦ ⟦ ρ ↦ ∅ ⟧.ρ.k, k ↦ ⟦⟧ ⟧.x ⟧}
$ eo-phi-normalizer rewrite -s bar.phi
{
  ⟦
    m ↦ ⊥.k
  ⟧
}
```

Why `⊥.k` is not rewritten to `⊥`? It should, through the `R_DD` rule.

**Comment by Anatolay** [comment]
At the moment, when user-defined rules are not specified, it uses a built-in set of rules which has not been updated.

```
$ eo-phi-normalizer rewrite -s bar.phi
{
  ⟦
    m ↦ ⊥.k
  ⟧
}

$ eo-phi-normalizer rewrite -s bar.phi -r eo-phi-normalizer/test/eo/phi/rules/new.yaml
{
  ⟦
    m ↦ ⊥
  ⟧
}
```

**Comment by yegor256** [comment]
@Anatolay can you please update the rules to the latest version and make a release?

**Comment by deemp** [comment]
@yegor256, this issue should be resolved in the v2.2.1. Please check it.

**Comment by yegor256** [comment]
@deemp works fine now, thanks!
